### DISCUSSION TITLE
Unreleased RSS memory after zooming in/out

### DISCUSSION THREAD

**Discussion by ghost** [fltk-rs/fltk-rs#1373] [General]
If I zoom in/out (Ctrl + or -) several times, it seems to be the case that Wayland surfaces are allocated and not released afterwards. I suspect this is a general design issue of upstream or the Wayland driver.

Upstream bug report https://github.com/fltk/fltk/issues/702

**Comment by MoAlyousef** [comment]
I can't test at the moment since my kde machine doesn't support wayland. The FLTK coredevs might be able to provide better input.

**Comment by ghost** [comment]
Closed by upstream as wontfix and works-as-designed

**Comment by ghost** [comment]
For the record, in a prolonged test I could not trigger any memory release in FLTK's Wayland driver. Whatever I did in the simple hello world button demo, it only grew in size. Other Wayland native toolkit demos I tried do not exhibit this behavior. Maybe FLTK's pages would be evicted with really high VM pressure.
